app/main.py

```diff
@@ -2,0 +1,25 @@
+import os
+
+
+def move_file(command: str) -> None:
+    if not command.startswith("mv "):
+        raise ValueError(
+            "Command must start with 'mv '."
+            "")
+    parts = command.split(maxsplit=2)
+    if len(parts) != 3:
+        raise ValueError(
+            "Invalid command format."
+        )
+    _, source, destination = parts
+    if not os.path.isfile(source):
+        raise FileNotFoundError(f"Source file not found: {source}")
+    if destination.endswith("/"):
+        os.makedirs(destination, exist_ok=True)
+        destination = os.path.join(destination, os.path.basename(source))
+    else:
+        dest_dir = os.path.dirname(destination)
+        if dest_dir:
+            os.makedirs(dest_dir, exist_ok=True)
```

Comment:
The code checks if the destination directory exists and creates it if it doesn't. This is a good practice to avoid errors when moving the file.